org.freedesktop.Sdk.Extension.texlive.yml

```diff
@@ -61,60 +61,60 @@ modules:
         sha256: 2990a8d275506c297b2239a1b4c5d9a9ec0d18cf12ff9a6a33924cf2e3838ed4
   - name: ghostscript
     config-opts:
-    - --enable-dynamic
     - --disable-cups
     - --disable-dbus
+    - --enable-dynamic
     - --disable-gtk
     - --with-drivers=FILES
     make-args:
     - so
     make-install-args:
-    - soinstall
     - install
+    - soinstall
     sources:
     - type: archive
       url: https://github.com/ArtifexSoftware/ghostpdl-downloads/releases/download/gs9533/ghostscript-9.53.3.tar.xz
       sha256: 9c9f5bc85b6c7eb08368c05b1e3339f7aaf9677ddca710c6283f872d55e2a234
     cleanup:
     - /share/doc
-    - /share/man
     - /share/ghostscript/9.53.3/doc
     - /share/ghostscript/9.53.3/examples
+    - /share/man    
   - name: asymptote
     config-opts:
       - --bindir=${FLATPAK_DEST}/bin/${FLATPAK_ARCH}-linux
-      - --with-latex=${FLATPAK_DEST}/texmf-dist/tex/latex
       - --with-context=${FLATPAK_DEST}/texmf-dist/tex/context/third
+      - --with-latex=${FLATPAK_DEST}/texmf-dist/tex/latex
     sources:
       - type: archive
         url: https://downloads.sourceforge.net/asymptote/asymptote-2.68.src.tgz
         sha256: e1e85a5db14dc809a43189f85415bd0845bcb9eec7aea5533767838d045b02b2
   - name: dvisvgm
     buildsystem: autotools
     build-options:
-      cxxflags: -I/usr/lib/sdk/texlive/include
       cflags: -I/usr/lib/sdk/texlive/include
+      cxxflags: -I/usr/lib/sdk/texlive/include
       ldflags: -L/usr/lib/sdk/texlive/lib
       env:
         PKG_CONFIG_PATH: /usr/lib/sdk/texlive/lib/pkgconfig
     config-opts:
       - --bindir=${FLATPAK_DEST}/bin/${FLATPAK_ARCH}-linux
-      - --enable-bundled-libs      
+      - --enable-bundled-libs
     sources:
       - type: archive
-        url: https://github.com/mgieseki/dvisvgm/releases/download/2.10.1/dvisvgm-2.10.1.tar.gz
-        md5: e1fa5d5ce656fedcdb7cd4146220e181
+        url: https://github.com/mgieseki/dvisvgm/archive/2.11.1.tar.gz
+        sha256: 71b772d97abe04bc0b9a4dbd0077f4224ade7ea0aac325f061e4c0037d31020c
   - name: clisp
     build-options:
       env:
-        CPPFLAGS: -I/usr/lib/sdk/texlive/include
-        LDFLAGS: -L/usr/lib/sdk/texlive/lib
+        cppflags: -I/usr/lib/sdk/texlive/include
+        ldflags: -L/usr/lib/sdk/texlive/lib
```

Comment:
These are environment variables. Hence the capitalization.

Reply:
Sorry, I was not aware of that.